sched/plan_class_spec.cpp

```diff
@@ -1286,12 +1286,14 @@ int main() {
     }
 
     for (unsigned int i=0; i<pcs.classes.size(); i++) {
-        bool b = pcs.check(sreq, pcs.classes[i].name, hu);
+        WORKUNIT wu;
```

Comment:
I don't understand this change. It looks like not related to this particular PR at all.
Could you please specify why this is needed and why you set constant `100` values to `wu.id` and `wu.batch` ?

Reply:
This fixed a compile error; PLAN_CLASS_SPEC::check() now takes a final WORKUNIT arg,
and this code (test code, normally ifdef'd out) was missing it.

Reply:
The values (100) are so that you can test the plan class features related to WU ID and batch number.